simple_geo3x3.php

```diff
@@ -1,9 +1,9 @@
 <?php
-require('Geo3x3.php');
+
+require __DIR__ . '/Geo3x3.php';
```

Comment:
Relative path is looked up from `include_path`, so we load them explicitly with `__DIR__`.